Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -14,6 +14,8 @@ import Alamofire
 import Account
 import ReadingList
 import MobileCoreServices
+import ObjectiveC
```

Comment:
its used for the fetching of the accessory view 
`let input = webContentView.performSelector(Selector("inputAccessoryView"))`